pydra/engine/core.py

```diff
@@ -677,7 +686,7 @@ def __init__(
 
         """
         if input_spec:
-            if isinstance(input_spec, BaseSpec):
+            if isinstance(input_spec, BaseSpec):  # TODO: test this block
```

Comment:
is this todo taken care of?

Reply:
CC: @nicolocin

Reply:
assuming that @nicolocin is dealing with it, removing todo